LICENSE

```diff
@@ -0,0 +1,176 @@
+                                 Apache License
```

Comment:
I added boilerplate text in the `README.md` that is common to other CARMA projects. One notable change I made was to make the attribution and license links specific to this repo. 

Other CARMA projects link to the CARMA Platform attribution and license files, even though there is a repo-specific `License.md`. I think license and attributions specific to the repo make more sense since not all CARMA projects will have the same set of developers and dependency licenses.